IFC4x3/Properties/n/NoiseLimitValue_02ksEi$yvANQhP$HONlrqo/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NoiseLimitValue_02ksEiyvANQhPHONlrqo" Name="NoiseLimitValue" UniqueId="02bb63ac-ffce-4a5d-aad9-fd1617bf5d32" PrimaryDataType="IfcSoundPowerMeasure" />
```

Comment:
IfcSoundPowerMeasure was renamed in IFC4 to IfcSoundPowerLevelMeasure if this property is retained.

Reply:
This concept doesn't realy fit with NoiseEmissions description. We are here considering the maximum noise emissions allowed. Should we modify the description of NoiseEmission ? (NoiseEmission by nature should be the maximum that is accepted?)  @SergejMuhic @czapplitec 
I have already change Primary Data type in case we keep NoiseLimitValue. commit : 75ce298398dad44c23e6578742e9c157bce6a214

Reply:
I would ask @AlexBrad1eyCT about this. Does it fir with the original intention of the property or would a new one be the preferable solution?